resources/js/components/SearchForm.vue

```diff
@@ -0,0 +1,161 @@
+<script>
+import Vue from 'vue';
+
+const moment = require('moment');
+require('moment/locale/de');
+import checkView from 'vue-check-view';
+Vue.use(checkView);
+
+export default {
+    name: 'SearchForm',
+    props: {
+        committees: {
+            type: Array,
+            default: () => []
+        }
+    },
+    data() {
+        return {
+            inputValue: '',
+            committeesList: this.committees,
+            sortedCommittees: [],
+            filtered: false
+        };
+    },
+    created () {
+        this.sortByChar();
+    },
+    methods: {
+        sortByChar() {
+            let char = '';
+            this.committeesList
+                .sort((a, b) => a.title === b.title ? 0 : +(a.title > b.title) || -1)
+                .forEach(el => {
+                    if (!this.sortedCommittees.length || char !== el.title[0].toLowerCase()) {
+                        char = el.title[0].toLowerCase();
+                        let arr = this.committeesList.filter(el => el.title[0].toLowerCase() === char);
+                        this.sortedCommittees.push({
+                            data: arr,
+                            char
+                        });
+                    }
+                });
+        },
+        filterList (e) {
+            let value = e.target.value;
+            if (value) {
+                this.sortedCommittees = this.committeesList
+                    .filter(el => el.title.toLowerCase().includes(value.toLowerCase()));
+                this.filtered = true;
+            }
+            else {
+                this.sortedCommittees = [];
+                this.sortByChar();
+                this.filtered = false;
+            }
+        },
+        scrollToComponents(id) {
+            const $el = document.querySelector(`#${id}-list-element`);
+            if ($el) {
+                $el.scrollIntoView({ behavior: 'smooth'});
+            }
+        },
+        viewHandler(e) {
+            if (e.percentInView === 1 || e.percentTop > .2 && e.percentTop < .8) {
+                document.querySelector(`#${e.target.element.id[0]}-search-button`).classList.add('bolt');
```

Comment:
check if the element is actually available on the page, like you did here https://github.com/nextsolx/ratsinformationssystem/blob/5a9614c21694b3f94c058c5edd144255a5224bb1/resources/js/components/SearchForm.vue#L58-L61